sharding/data_modeling.rst

```diff
@@ -0,0 +1,222 @@
+.. _distributed_data_modeling:
+
+Distributed modeling refers to choosing how to distribute information across nodes in a multi-machine database cluster and query it efficiently. There are common use cases for a distributed database with well understood design tradeoffs. It will be helpful for you to identify whether your application falls into one of these categories in order to know what features and performance to expect.
+
+Citus uses a column in each table to determine how to allocate its rows among the available shards. In particular, as data is loaded into the table, Citus uses this so-called *distribution column* as a hash key to allocate each incoming row to a shard.
+
+The database administrator, not Citus, picks the distribution column of each table. Thus the main task in distributed modeling is choosing the best division of tables and their distribution columns to fit the queries required by an application.
```

Comment:
I added this flourish because of a question I heard in a tech-fit. The person wondered if they could pick the column or if it was out of their control. Samay assured me this is a common question. However "The database administrator picks..." does answer the question more succinctly.